### DISCUSSION TITLE
Cant connect to the internet via mobile network

### DISCUSSION THREAD

**Discussion by 5071177** [wg-easy/wg-easy#923] [Q&A]
Hi there. 
WG easy works just fine over wifi but doesnt work over mobile network with the same phone. 
Any ideas how to fix? 

ps I am a total noob so I can ask stupid questions, sorry :)

**Comment by vithurshanselvarajah** [comment]
I think we may be lacking some context here. 

Can you explain your current configuration a bit better -

Where are you hosting wg-easy? 

If its hosted from home have you configure port forwarding? 

Does it work on any wifi or only a particular one?

**Reply by BrandonMFong** [reply]
I'm experiencing this. 

1. I'm hosting my wg-easy on Truenas Scale. 
2. I have port forwarding setup on my xfinity router. 
3. I currently using the VPN under one AT&T wifi. Haven't tried other wifis. I'm migrating from PiVPN to wg-easy.

**Comment by dimonique** [comment]
Same problem here. Using Ihor VPS server (London Datacenter). Using different local CIS operators. VPN via WiFi works fine and it doesn't work when im using mobile internet. Any suggestions?
